definitions/BiG_DRP.def

```diff
@@ -0,0 +1,52 @@
+Bootstrap: docker
+From: pytorch/pytorch:1.12.0-cuda11.3-cudnn8-devel
+
+%labels
+	MANTAINER Rohan Gnanaolivu
+
+%setup
+	cp ./src/Singularity_gpu_fix.sh $SINGULARITY_ROOTFS
+	# add local url of this repository for testing
+
+
+%environment
+	PATH=$PATH:/usr/local/BiG-DRP
+	MODEL_DIR=/usr/local/BiG-DRP
```

Comment:
It should be IMPROVE_MODEL_DIR, which has to match the variable inside the train.sh or preprocess.sh scripts